.github/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 2. Enhancement - Added Derby (EGBD) SMR - thanks to @rishab-alt
 3. Enhancement - Removed Manchester (EGCC) disused stands - thanks to @rishab-alt
 4. Procedure Change (2309) - Amended EuroCenter vACC EURM-W and EURW-N callsigns to EUC-MW and EUC-WN, respectively - thanks to @hsugden (Harry Sugden)
-
+5. AIRAC (2309) Remove Bristol (EGGD) stands 31R & 33L - thanks to @rishab-alt
 # Changes from release 2023/07 to 2023/08
```

Comment:
```suggestion
5. AIRAC (2309) - Removed Bristol (EGGD) stands 31R & 33L - thanks to @rishab-alt

# Changes from release 2023/07 to 2023/08
```

Reply:
ah , ok , wasnt sure if it was remove remove , ill do that , sorry